src/jitlog/test.lua

```diff
@@ -147,15 +147,36 @@ local function checkheader(header)
   assert(header.version > 0)
 end
 
+local statsmixin = {
+  init = function(self)
+    local datatotals = {}
+    for i = 0, 255 do
+      datatotals[i] = 0
+    end
+    self.datatotals = datatotals
+  
+    local msgcounts = {}
+    for i = 0, 255 do
+      msgcounts[i] = 0
+    end
+    self.msgcounts = msgcounts
+  end,
+  aftermsg = function(self, msgtype, size, pos)
+    self.datatotals[msgtype] = self.datatotals[msgtype] + size
+    self.msgcounts[msgtype] = self.msgcounts[msgtype] + 1
+  end,
+}
+
+local testmixins = {
```

Comment:
This seems a bit superfluous given that it only contains one entry?

Reply:
Well the API is designed to be passed a list of mixins

Reply:
Yes, but then it doesn't need to be an intermediate variable does it?